### PR TITLE
new package: skippy-xd

### PR DESCRIPTION
**Pull request by antonio-malcolm**

### CODE REVIEW

File: srcpkgs/skippy-xd/template [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+	freetype-devel imlib2-devel fontconfig-devel giflib-devel libjpeg-turbo-devel libpng-devel zlib-devel"
+short_desc="An Expose-like full-screen task-switcher for X11"
+maintainer="Antonio Malcolm <antonio@antoniomalcolm.com>"
+license="GPL-2"
+homepage="https://github.com/antonio-malcolm/skippy-xd"
+distfiles="${homepage}/archive/v2015.12.21.tar.gz"
+checksum=313f93abaf98007dd6a8c1942bcaab16d418c5f6d4625f3d82115822ac1d3706
+
+post_install() {
+	vlicense COPYING
```
Comment (xtraeme) [inline srcpkgs/skippy-xd/template]: `GPL` licenses do not need to be installed.

File: srcpkgs/skippy-xd/template [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+build_style=gnu-makefile
+hostmakedepends="pkg-config"
+makedepends="libX11-devel libXext-devel libXft-devel libXcomposite-devel libXdamage-devel
+	libXext-devel libXfixes-devel libXinerama-devel libXmu-devel libXrender-devel
+	freetype-devel imlib2-devel fontconfig-devel giflib-devel libjpeg-turbo-devel libpng-devel zlib-devel"
+short_desc="An Expose-like full-screen task-switcher for X11"
+maintainer="Antonio Malcolm <antonio@antoniomalcolm.com>"
+license="GPL-2"
+homepage="https://github.com/antonio-malcolm/skippy-xd"
+distfiles="${homepage}/archive/v2015.12.21.tar.gz"
```
Comment (xtraeme) [inline srcpkgs/skippy-xd/template]: Use `$version` instead.

**Comment by antonio-malcolm** [comment]
Ok, both adjustments are in.

File: srcpkgs/skippy-xd/template [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# Template file for 'skippy-xd'
+pkgname=skippy-xd
+version=2015.12.21
+revision=1
+build_style=gnu-makefile
+hostmakedepends="pkg-config"
+makedepends="libX11-devel libXext-devel libXft-devel libXcomposite-devel libXdamage-devel
+	libXext-devel libXfixes-devel libXinerama-devel libXmu-devel libXrender-devel
+	freetype-devel imlib2-devel fontconfig-devel giflib-devel libjpeg-turbo-devel libpng-devel zlib-devel"
```
Comment (xtraeme) [inline srcpkgs/skippy-xd/template]: Too many redundant dependencies: libXext-devel set twice, libX11-devel, etc.

**Comment by antonio-malcolm** [comment]
Ok, removed the singluar duplicated dependency.

**Comment by xtraeme** [comment]
Please squash those commits with a duplicate commit log into one to get this merged.

Thanks.

**Comment by antonio-malcolm** [comment]
Ok, commits squashed.

**Comment by xtraeme** [comment]
You are missing `conf_files="/etc/xdg/*.rc"`

**Comment by antonio-malcolm** [comment]
Added `conf_files="/etc/xdg/*.rc"` to the template.

**Pull request merged**